build/lib/electron.js

```diff
@@ -99,6 +99,9 @@ function getElectron(arch) {
     };
 }
 async function main(arch = process.arch) {
```

Comment:
How about this instead?

```suggestion
 async function main(arch = process.env["npm_config_arch"] || process.arch) {
```